src/main/java/com/diffblue/demo/ecommerce/controllers/CheckoutController.java

```diff
@@ -1,5 +1,6 @@
 package com.diffblue.demo.ecommerce.controllers;
 
+import com.diffblue.demo.ecommerce.Application;
 import com.diffblue.demo.ecommerce.forms.AddressForm;
 // Copyright 2016-2018 DiffBlue limited. All rights reserved.
```

Comment:
`Diffblue Limited` (all files)

Reply:
@peterschrammel Copyright will be fixed in a separate PR.